ctk/features/debug.feature

```diff
@@ -0,0 +1,42 @@
+Feature: Debug Task
+  As an implementer of the workflow DSL
+  I want to ensure that debug tasks can be executed within the workflow
+  So that my implementation conforms to the expected behaviour
+
+  # Tests Debug with literal constant
+  Scenario: Debug with literal constant
+    Given a workflow with definition:
+    """yaml
+    document:
+      dsl: 1.0.0-alpha1
+      namespace: default
+      name: debug
+    do:
+      printMessage:
+        debug: Hello world!
+    """
+    And given the workflow input is:
+    """yaml
+    """
+    When the workflow is executed
+    Then the workflow should complete
+
+  # Tests Debug with expression
+  Scenario: Debug with expression
+    Given a workflow with definition:
+    """yaml
+    document:
+      dsl: 1.0.0-alpha1
+      namespace: default
+      name: debug
+    do:
+      printMessage:
+        debug: ${Hello \(.name)!} 
+    """
+    And given the workflow input is:
+    """yaml
+      name: John
+    """
+    When the workflow is executed
+    Then the workflow should complete
```

Comment:
I think you can verify the output here. Should be `{ name: "John"}`.

Reply:
How do I verfy that something is printed to the console

Reply:
Not the console, but the output of this workflow. Shouldn't be the input in this context?

Reply:
Yes, but that check is not relevant for this test, I think, unless there is way to check what is printed, these test end condition should be that the workflow ends.

Reply:
It's a nit; but I think we should add it.